### PR TITLE
Change build yml for improve readability of tests

### PR DESCRIPTION
**Pull request by DonJayamanne**
Now its easy to tell what tests fail (local vs remote ....etc)

**Before**
image
**After**
image

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -618,17 +617,14 @@ jobs:
         uses: actions/setup-dotnet@v1.8.0
         with:
           dotnet-version: ${{env.DOTNET_VERSION}}
-        if: (matrix.test-suite == 'integration' || matrix.test-suite == 'notebook')
```
Comment (rchiodo) [inline .github/workflows/build-test.yml:621]: So these run on all tests now?

Comment (rchiodo) [reply .github/workflows/build-test.yml:621]: Oh I guess this was redundant.

**Review by rchiodo — APPROVED** [review]

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (DonJayamanne) [reply .github/workflows/build-test.yml:621]: Oh crap, i remvoed the wrong thing, damn it

Comment (DonJayamanne) [reply .github/workflows/build-test.yml]: Fixed

**Review by IanMatthewHuff — APPROVED** [review]

**Pull request merged**